cloud/skytap/virtualclassroom.rb

```diff
@@ -29,7 +29,7 @@ def create_base_environment(name, base_template)
   env_config = {
     'template_id' => base_template, 
     'name' => name,
-    'suspend_on_idle' => null
+    'suspend_on_idle' => 'null'
```

Comment:
After looking at the API I actually think we should just set this to the maximum of 4 days.  That would cover all of our use cases and prevent skytap envs from running forever.
http://help.skytap.com/API_Documentation.html?Highlight=suspend%20on%20idle#Environmentsresourcemodel